lib/ret_web.ex

```diff
@@ -34,7 +34,7 @@ defmodule RetWeb do
         namespace: RetWeb
 
       # Import convenience functions from controllers
-      import Phoenix.Controller, only: [get_flash: 2, view_module: 1]
+      import Phoenix.Controller, only: [get_flash: 2]
```

Comment:
I see this is called out specifically to check in the TODOs but I don't see any mention of this in the upgrade notes. Whats this change do?

Reply:
This change makes it so that `Phoenix.Controller.view_module/1` is not automatically accessible from all the views (modules with `use RetWeb, :view`). 

First I made the changes as per this line [source](https://github.com/phoenixframework/phoenix/blob/v1.5/CHANGELOG.md#deprecations):
> [Layout] Use <%= @inner_content %> instead of <%= render @view_module, @view_template, assigns %> for rendering the child layout

Then I noticed that `view_module` was never used anywhere else in the project. I figured it was safe (and maybe desirable) for us to remove it, but I don't feel strongly about it.